src/cart/crt_init.c

```diff
@@ -309,14 +309,14 @@ crt_init_opt(crt_group_id_t grpid, uint32_t flags, crt_init_options_t *opt)
 	struct timeval	now;
 	unsigned int	seed;
 	const char	*path;
-	bool		server;
+	bool		server = false;
```

Comment:
This isn't needed.

Reply:
yeah it was to retrigger build that failed last time due to unrelated nlt failure

Reply:
Everything other than the server should be running under valgrind in NLT, was it the server that was affected?